### PR TITLE
Update README.md 

### PR DESCRIPTION
**Pull request by Moejoe90**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,149 @@
...
+## Adding your own ICOs to the Transparency-Monitor
+The ICO-monitor collects information from the blockchain public-ledger using a set of pre-defined
+rules set in [\config.js](https://github.com/Neufund/ico-transparency-monitor/blob/master/src/config.js)
+
+To add your own ICO you would have to
+
+1 - Include the ICO Smart-Contract [ABI](https://github.com/ethereum/wiki/wiki/Ethereum-Contract-ABI) in the [\Smart_Contracts](https://github.com/Neufund/ico-transparency-monitor/tree/master/src/smart_contracts) folder
+    this ABI is generated from the smart-contract source code and can be found in some cases in [etherscan](https://etherscan.io/)
+
+2 - Configure [\config.js](https://github.com/Neufund/ico-transparency-monitor/blob/master/src/config.js)
```
Comment (banciur) [inline README.md]: he moe comment

Comment (Moejoe90) [inline README.md]: Thanks for the comment

File: README.md [hunk 2]
```diff
@@ -1,7 +1,106 @@
-# ico-transparency-monitor
+# ICO-Transparency-Monitor
 
 [![Build Status](https://travis-ci.org/Neufund/ico-transparency-monitor.svg)](https://travis-ci.org/Neufund/ico-transparency-monitor) [![Greenkeeper badge](https://badges.greenkeeper.io/Neufund/generic-ico.svg)](https://greenkeeper.io/)
 
+## Adding costume ICOs to the Transparency-Monitor
```
Comment (banciur) [inline README.md]: typo `costume`

File: README.md [hunk 3]
```diff
@@ -1,7 +1,106 @@
...
+The ICO-monitor collects information from the blockchain public-ledger using a set of pre-defined
+rules set in [config.js](https://github.com/Neufund/ico-transparency-monitor/blob/master/src/config.js)
+
+To add your own ICO you would have to
+
+1 - Include the ICO Smart-Contract [ABI](https://github.com/ethereum/wiki/wiki/Ethereum-Contract-ABI) in the [Smart_Contracts](https://github.com/Neufund/ico-transparency-monitor/tree/master/src/smart_contracts) folder
```
Comment (banciur) [inline README.md]: I would use markdown [numbered lists ](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists) syntax

**Pull request merged**